css/_filmstrip.scss

```diff
@@ -48,21 +48,64 @@
     &__videos {
         @extend %align-right;
         position:relative;
-        height:196px;
         padding: 0;
         /* The filmstrip should not be covered by the left toolbar. */
         bottom: 0;
         width:auto;
         transition: bottom 2s;
         overflow: visible !important;
-        /*!!! Removes the gap between the local video container and the remote
-        videos. */
-        font-size: 0pt;
 
         &#remoteVideos {
             border: $thumbnailsBorder solid transparent;
             padding-left: $defaultToolbarSize + 5;
-       }
+        }
+
+        /**
+         * The local video identifier.
+         */
+        &#filmstripLocalVideo {
+            bottom: 32px;
+            flex-direction: column;
+
+            /**
+             * The invite button style.
+             */
+            .filmstrip__invite {
+                padding-bottom: 5px;
+                z-index: $dropdownZ;
```

Comment:
In horizontal filmstrip, local video tooltips and stats menu display below the invite button. I'm not sure what z-index wars will need to be fought to fix it...

Reply:
Okay, so the battle is with .hide-videos setting a z-index that's too low and removing the z-index might be a naive yet effective fix. I can't remember immediately why the lower z-index is needed, maybe to prevent the label from being unclickable when docked to the top right of the screen. Regardless I doubt the z-index is needed now because the interaction between video label and filmstrip is much simpler with the local video on top.